### PR TITLE
Add version locked kairos deposit contract and contract client.

### PR DESCRIPTION
**Pull request by jonas089**
See kairos-deposit-contract/readme.md for instructions on how to compile the .wasm binaries and run the tests against the execution engine. All dependencies on casper-node (including casper-types and the EE) are version locked to jonas089/casper-node --rev "0bcc25f4a203be4e78eed679b54127ab1ed848ab"

The kairos-contract-cli can be used as a binary or library to 
- call entry points of the deposit contract
- deploy session code for deposit / withdrawal requests

The deposit contract stores deposit events in a dictionary, withdrawal events are not stored (yet).

### CODE REVIEW

File: kairos-deposit-contract/readme.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ For now, on Macos:
 
 or, on Ubuntu/Debian:
 
-`./build-ubuntu-feat-2.0.sh`
+`./build-ubuntu-feat-2.0.sh`
```
Comment (marijanp) [inline kairos-deposit-contract/readme.md:97]: This is not serving the purpose adding a space to sign this specific commit. All the other commits are still unsigned. please read https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits and configure your git accordingly

**Review by marijanp — CHANGES_REQUESTED** [review]
Removing the DS_Store file should not be part of the git history moreover commiting a DS_Store file should never be done in the first place. I would suggest you configure your git accordingly and rewrite history

**Comment by marijanp** [comment]
You can drop the sign commit [b8ed071](https://github.com/cspr-rad/kairos/pull/24/commits/b8ed071302af1dfbf6fd3fe383387c269a5788e7)

**Comment by marijanp** [comment]
There is still a top-level DS_Store

**Comment by jonas089** [comment]
> There is still a top-level DS_Store

I'll have to check where it comes from and get rid of it.

**Pull request closed without merging**